proxies/live/apiproxy/policies/RaiseFault.InvalidTargetIdentifier.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<RaiseFault async="false" continueOnError="false" enabled="true" name="RaiseFault.InvalidTargetIdentifier">
+  <DisplayName>RaiseFault.InvalidTargetIdentifier</DisplayName>
+  <Properties/>
+  <FaultResponse>
+    <Set>
```

Comment:
Does the assign message end up overriding the setting of the payload/status code and reason? I guess it must.

Reply:
Yes, the latest step will override any previous ones